src/main/steps/applicant1/hub-page/joint/latest-update-content/awaiting-final-order.njk

```diff
@@ -1 +1 @@
-<p class="govuk-body">{{ awaitingFinalOrder.line1 }}</p>
+{% extends "../../sole/latest-update-content/awaiting-final-order-or-final-order-overdue.njk" %}
```

Comment:
why have this file? why not point to the one youre extending.

Reply:
Yes I made changes to this file, it now points to the content in the applicant1/hub-page/joint/content.ts file.